### ISSUE TITLE
Pass array in RequestParam

### ISSUE THREAD

**Issue by pjalinski** [lagoshny/ngx-hateoas-client#44]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

```diff
diff --git a/node_modules/@lagoshny/ngx-hateoas-client/esm2020/lib/util/url.utils.mjs b/node_modules/@lagoshny/ngx-hateoas-client/esm2020/lib/util/url.utils.mjs
index 1cbb6b3..19ea7e5 100644
--- a/node_modules/@lagoshny/ngx-hateoas-client/esm2020/lib/util/url.utils.mjs
+++ b/node_modules/@lagoshny/ngx-hateoas-client/esm2020/lib/util/url.utils.mjs
@@ -25,8 +25,14 @@ export class UrlUtils {
                         resultParams = resultParams.append(key, value.getSelfLinkHref());
                     }
                     else {
+                        if (options.params[key] instanceof Array) {
+                        options.params[key].forEach((item) => {
+                          resultParams = resultParams.append(`${key.toString()}`, item);
+                        });
+                      } else {
                         // Else append simple param as is
                         resultParams = resultParams.append(key, value.toString());
+                      }
                     }
                 }
             }
diff --git a/node_modules/@lagoshny/ngx-hateoas-client/fesm2015/lagoshny-ngx-hateoas-client.mjs b/node_modules/@lagoshny/ngx-hateoas-client/fesm2015/lagoshny-ngx-hateoas-client.mjs
index 0201bab..17bdde9 100644
--- a/node_modules/@lagoshny/ngx-hateoas-client/fesm2015/lagoshny-ngx-hateoas-client.mjs
+++ b/node_modules/@lagoshny/ngx-hateoas-client/fesm2015/lagoshny-ngx-hateoas-client.mjs
@@ -406,8 +406,14 @@ class UrlUtils {
                         resultParams = resultParams.append(key, value.getSelfLinkHref());
                     }
                     else {
+                      if (options.params[key] instanceof Array) {
+                        options.params[key].forEach((item) => {
+                          resultParams = resultParams.append(`${key.toString()}`, item);
+                        });
+                      } else {
                         // Else append simple param as is
                         resultParams = resultParams.append(key, value.toString());
+                      }
                     }
                 }
             }
diff --git a/node_modules/@lagoshny/ngx-hateoas-client/fesm2020/lagoshny-ngx-hateoas-client.mjs b/node_modules/@lagoshny/ngx-hateoas-client/fesm2020/lagoshny-ngx-hateoas-client.mjs
index 9b9fc77..565ade3 100644
--- a/node_modules/@lagoshny/ngx-hateoas-client/fesm2020/lagoshny-ngx-hateoas-client.mjs
+++ b/node_modules/@lagoshny/ngx-hateoas-client/fesm2020/lagoshny-ngx-hateoas-client.mjs
@@ -409,8 +409,14 @@ class UrlUtils {
                         resultParams = resultParams.append(key, value.getSelfLinkHref());
                     }
                     else {
+                      if (options.params[key] instanceof Array) {
+                        options.params[key].forEach((item) => {
+                          resultParams = resultParams.append(`${key.toString()}`, item);
+                        });
+                      } else {
                         // Else append simple param as is
                         resultParams = resultParams.append(key, value.toString());
+                      }
                     }
                 }
             }
diff --git a/node_modules/@lagoshny/ngx-hateoas-client/lib/model/declarations.d.ts b/node_modules/@lagoshny/ngx-hateoas-client/lib/model/declarations.d.ts
index 71e931b..c580014 100644
--- a/node_modules/@lagoshny/ngx-hateoas-client/lib/model/declarations.d.ts
+++ b/node_modules/@lagoshny/ngx-hateoas-client/lib/model/declarations.d.ts
@@ -79,7 +79,7 @@ export interface GetOption extends RequestOption {
  * otherwise exception will be thrown.
  */
 export interface RequestParam {
-    [paramName: string]: Resource | string | number | boolean;
+    [paramName: string]: Resource | string | number | boolean| Array<string> | Array<number>;
 }
 /**
  * Page content params.
```

**Comment by lagoshny** [comment]
@pjalinski ,
Fixed in new versions for old Angular 2.3.9 and for the last Angular 3.0.9.

**Comment by pjalinski** [comment]
in commit daf2d8f5be38c71422c5dae84c4c5bc0893db82e

```
          } else if (isArray(options.params[key])) {
            (options.params[key] as Array<any>).forEach((item) => {
              console.log(`${ key.toString() } - ${item}`);
            });
```
you have console.log :)

**Comment by lagoshny** [comment]
@pjalinski thanks! 
Fix it in new versions for old Angular 2.3.10 and for the last Angular 3.0.10.
